osquery/dispatcher/scheduler.h

```diff
@@ -43,6 +44,45 @@ class SchedulerRunner : public InternalRunnable {
   std::chrono::milliseconds getCurrentTimeDrift() const noexcept;
 
  private:
+   inline void calculateTimeDrift(std::chrono::time_point<std::chrono::steady_clock> start_time_point) {
```

Comment:
It's look a bit messy to define start time point in one function, but end time point in another.  I'd suggest to calculated `loop_step_duration` also in `start` method and pass it to `calculateTimeDrift`. What do you think?